libs/api.lua

```diff
@@ -136,29 +200,32 @@ return function (prefix)
       return next(authors) and authors
     end,
     "^/search/(.*)$", function (query)
+      -- escape all special pattern characters except . and *
+      query = query:gsub("[%(%)%%%+%-%?%[%]%^%$]", function(c) return "%" .. c end)
       local matches = {}
+      local authorQuery, separator, packageQuery = query:match("^(.-)(/?)([^/]*)$")
```

Comment:
I guess this will work.  It puts the extra path segments in with the author.  That's probably the desired behavior.

For example, I might want to put the weblit-* packages under "creationix/weblit/*"